python-libraries/data-platform-catalogue/data_platform_catalogue/entities.py

```diff
@@ -50,22 +66,41 @@ def from_data_product_metadata_dict(metadata: dict, version, owner_id: str):
             description=metadata["description"],
             version=version,
             owner=owner_id,
+            owner_display_name=metadata["dataProductOwnerDisplayName"],
```

Comment:
`metadata.get("dataProductOwnerDisplayName")`

Reply:
I haven't done a get call with mandatory field. We could add verification that mandatory fields are filled?

Reply:
I think this is probably fine for now, if we call it without the mandatory fields we will get a KeyError and it should be easy to debug.